files/reframe

```diff
@@ -1,3 +1,3 @@
-REFRAME_GENERAL username=%{USERNAME_PAT}\|version=%{DATA:version}\|name=%{DATA:name}\|system=%{DATA:system}\|partition=%{DATA:partition}\|environ=%{DATA:environ}\|num_tasks=%{INT:num_tasks:int}\|tags=%{DATA:tags}\|jobid=%{DATA:jobid}
+REFRAME_GENERAL username=%{USERNAME_PAT}\|version=%{DATA:version}\|commit=%{DATA:commit}\|name=%{DATA:name}\|system=%{DATA:system}\|partition=%{DATA:partition}\|environ=%{DATA:environ}\|num_tasks=%{INT:num_tasks:int}\|num_tasks_per_node=%{INT:num_tasks_per_node:int}\|tags=%{DATA:tags}\|jobid=%{DATA:jobid}
```

Comment:
```suggestion
REFRAME_GENERAL username=%{USERNAME_PAT}\|version=%{DATA:version}\|(?:commit=%{DATA:commit}\|)?name=%{DATA:name}\|system=%{DATA:system}\|partition=%{DATA:partition}\|environ=%{DATA:environ}\|num_tasks=%{INT:num_tasks:int}\|num_tasks_per_node=%{INT:num_tasks_per_node:int}\|tags=%{DATA:tags}\|jobid=%{DATA:jobid}
```

Reply:
and leave at least one test with the old pattern

Reply:
why not just (...)? instead of (?:...)?
I tested it in ES and it gives exactly the same result

Reply:
a capture vs a non-capturing group.

There is no point in capturing this group? Only the commit itself?